Frontend/shared/src/hooks/AuthContext/index.ts

```diff
@@ -0,0 +1,2 @@
+export { AuthProvider, useAuth } from "./AuthContext";
+export {AuthContextType} from './AuthContextType';
```

Comment:
Add space before and after `AuthContextType` within the braces (like the above export has).